compiler/src/dotty/tools/dotc/transform/Recheck.scala

```diff
@@ -197,6 +194,7 @@ abstract class Recheck extends Phase, SymTransformer:
     def reset()(using Context): Unit =
       for (ref, mbr) <- prevSelDenots.iterator do
         ref.withDenot(mbr)
+      preRecheckPhase
```

Comment:
Is this line accidental?  Otherwise, I can't tell what this is forcing - perhaps it's worth commenting?

Reply:
Yes, that looks like an oversight.